routes/registration.js

```diff
@@ -1,36 +0,0 @@
-var express = require('express');
```

Comment:
Registration is the same as creating a user. So I moved the code to user routes instead.